### PR TITLE
 feature/TK-456

### PR DESCRIPTION
**Pull request by ani2fun**
Frontend Select Component to add personal user event

### CODE REVIEW

**Comment by nicmarti** [comment]
We're waiting for PR #332 related to TK-452 before we can merge this PR

File: frontend/src/components/Events/NewEventTemplateForm.js
Comment (Ven0mX) [inline frontend/src/components/Events/NewEventTemplateForm.js:214]: Why did you swap the date and year?

**Review by Ven0mX — APPROVED** [review]
LGTM, but I let you a comment

File: frontend/src/components/Events/NewEventTemplateForm.js
Comment (ani2fun) [reply frontend/src/components/Events/NewEventTemplateForm.js:214]: easy to read in front end and follow DD-MM-YYYY convention

**Pull request merged**
